stdlib/2/urllib2.pyi

```diff
@@ -45,14 +56,21 @@ class Request(object):
     def get_header(self, header_name: str, default: Optional[str] = ...) -> str: ...
     def header_items(self): ...
 
+class _Response(_SupportsRead, metaclass=ABCMeta):
+    def geturl(self) -> str: ...
+    def info(self) -> Mapping[str, str]: ...
+    def getcode(self) -> Optional[int]: ...
+
 class OpenerDirector(object):
+    addheaders: List[Tuple[str, str]] = ...
```

Comment:
Remove the `= ...`.

Reply:
Done in b994fd5d89edf5bd7b3a2d3dee76b0529209f24e